components/tools/OmeroJava/test/integration/ScriptServiceTest.java

```diff
@@ -245,12 +269,40 @@ public void testUploadOfficialLUTAsRoot() throws Exception {
         IScriptPrx svc = factory.getScriptService();
         List<OriginalFile> scripts = svc.getScriptsByMimetype(LUT_MIMETYPE);
         int n = scripts.size();
-        try {
-            long id = svc.uploadOfficialScript(folder, buf.toString());
-            Assert.assertTrue(id > 0);
-        } catch (Exception e) {
-        }
-        Assert.assertEquals(n+1, svc.getScriptsByMimetype(LUT_MIMETYPE).size());
+        long id = svc.uploadOfficialScript(folder, buf.toString());
```

Comment:
Not introduced by the PR, but instead of `buf.toString()` just use `"abc"`? The StringBuffer and for loop seems to be a very complex way just to generate the String `abc`.